tests/robot-tests/scripts/visual-testing/generate_permalink_test_cases.py

```diff
@@ -0,0 +1,41 @@
+from logging import warning
+import argparse
+import os
+import csv
+
+ap = argparse.ArgumentParser()
+ap.add_argument("-f", "--file", required=True,
+                help="the permalink urls csv to use to generate the robot test script from the template")
+ap.add_argument("-t", "--target", required=True,
+                help="the target filename of the generated robot test script")
+args = vars(ap.parse_args())
+
+permalinks_csv_filepath = args['file']
+
+permalink_ids = []
+
+with open(permalinks_csv_filepath, 'r', encoding='utf-8-sig') as csv_file:
+    csv_reader = csv.reader(csv_file, delimiter=',')
+    next(csv_reader)
+    for row in csv_reader:
+        permalink_ids.append(row[0])
+
+robot_tests_folder = '../../tests/visual_testing'
```

Comment:
Just wondering if we'll run into path issues here for people using windows?

Reply:
Possibly - I'd need a victim to try it in git bash...

Reply:
I'll take a look today

Reply:
Hey Luke, cheers for checking this out.  I couldn't reproduce this until I cleared down all my virtualenvs and started from scratch, and then I was able to reproduce.

Created a new lockfile which is working for me if you could check it again please?  Also updated locations where snapshots are taken so as not to junk up people's systems with snapshot folders during test runs.

And finally also saw that one of the seed data csv files was empty - been getting that a few times on this branch so I'm not 100% sure what's going on there.